### ISSUE TITLE
Issue with (and question about) the Wiki

### ISSUE THREAD

**Issue by btg611** [MagnumMacKivler/RLCPT2#14]
According to the Wiki, the Baldwin VO-1000 was a proposed locomotive that was never produced, but Baldwin actually produced over 500 of them... 548 to be exact!

Having said this, I'm curious if there is a way to publically contribute to the wiki, seeing how much information is still missing. I have a decent amount of resources with (to the best of my knowledge) accurate information on a wide variety of locomotives and I'm more than willing to share it if it helps.

**Comment by IJARika** [comment]
you'll notice that it says "VO-1000 (proposed VO _**road switcher**_)", the normal switcher isn't marked as a paper engine.

**Comment by mrlazorz** [comment]
Baldwin is notorious for redundant designations; the VO-1000 _switcher_ was indeed mass produced, but the road switcher did not see sales or production before the 600 series engine became available.

Wiki access is usually restricted to prevent defacing or those unfamiliar with some of PT's quirks regarding real-life specs (e.g.: transitions), but you're welcome to message staff about it - seeing as i'm currently modelling baldwin and have a few spec books feel free to message me about contributions on discord `@Laz#2048`.

**Comment by btg611** [comment]
Yes, I see what you guys mean now... LH was notorious for the redundancy in designations too, it seems. I was not aware Baldwin had proposed more than one VO switcher, so thanks for pointing that out.

Laz, thanks for that. I'll compile some info on a few locos I've found particularly interesting in the coming week or so and get back to you. Thanks for the models you've put out thus far. I look forward to your Baldwin contributions (we've certainly been lacking on them) and honestly envy your modeling skills.. I've dabbled but was never any good!
